documentation/modules/auxiliary/admin/sap/sap_igs_xxe.md

```diff
@@ -20,33 +27,30 @@ Once set up and configured, the instances will be vulnerable on the default HTTP
 
 ## Verification Steps
 
-  1. Install the module as usual
-  2. Start msfconsole
-  3. Do: `workspace [WORKSPACE]`
-  4. Do: `use auxiliary/admin/sap/sap_igs_xxe`
-  5. Do: `set RHOSTS [IP]`
-  6. Do: `set FILE [remote file name]`
-  7. Do: `set SHOW [true|false]`
-  8. Do: `set action READ`
-  9. Do: `check`
-  10. Do: `run`
+  1. Start msfconsole
+  1. Do: `workspace [WORKSPACE]`
+  1. Do: `use auxiliary/admin/sap/sap_igs_xxe`
+  1. Do: `set RHOSTS [IP]`
+  1. Do: `set FILE [remote file name]`
+  1. Do: `set SHOW [true|false]`
+  1. Do: `set action READ`
+  1. Do: `check`
+  1. Do: `run`
+  1. Verify that the contents of the file you specified were returned.
 
 ## Options
 
 ### FILE
 
-File to read from the remote server, example: `/etc/passwd`
-
-### SHOW
-
-`true` - show remote file content and save in workspace loot
-
-`false` - not show remote file content and save in workspace loot
+File to read from the remote server. Example: `/etc/passwd`
 
 ### URN
 
-XXE vulnerable SAP IGS URN: `/XMLCHART` - generates business graphics with XML-based customizing and
-XML- based data (BI 7.x Chart Item, BSP, WD)
+This is the URL of the XMLCHART page on the SAP IGS server that is vulnerable to XXE.
```

Comment:
URI = URL + URN
URI: `http://10.10.10.10/XMLCHART`
URL: `http://10.10.10.10`
URN: `/XMLCHART`

`/XMLCHART` is not URL
`/XMLCHART` is URN

Reply:
"URNs were distinguished from URLs, which identify resources by specifying their locations in the context of a particular access protocol, such as HTTP or FTP. In contrast, URNs were conceived as persistent, location-independent identifiers assigned within defined namespaces, typically by an authority responsible for the namespace, so that they are globally unique and persistent over long periods of time, even after the resource which they identify ceases to exist or becomes unavailable." - Taken from https://en.wikipedia.org/wiki/Uniform_Resource_Name#URIs,_URNs,_and_URLs

Looking at the definition for URI we can see that /XMLCHART in this context would be more consistent with the definition of a `path` within a URI. See https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Generic_syntax

URNs are a bit different as they were designed to be "URNs were conceived as persistent, location-independent identifiers assigned within defined namespaces, typically by an authority responsible for the namespace, so that they are globally unique and persistent over long periods of time, even after the resource which they identify ceases to exist or becomes unavailable" according to https://en.wikipedia.org/wiki/Uniform_Resource_Name#URIs,_URNs,_and_URLs, so this is why I don't think the name URN is as appropriate here given that description doesn't appropriately describe what we are using `/XMLCHART` for here.

Reply:
Then the `URN` parameter can be renamed to `PATH`

Reply:
Yep happy to do that, realize I also messed up on my description here. I'll go ahead and edit this up so long so we rename it to `PATH` and will update the documentation and descriptions accordingly where needed.

Reply:
Will you change it or will I change it?

Reply:
I changed it: https://github.com/rapid7/metasploit-framework/pull/14163/commits/a17072dfa90e03dde528eede21ef2b531ba9e3b3

Reply:
Sorry have meetings this morning was planning to do them as soon as they ended, but I see you went around and fixed them already so all good

Reply:
Resolved as mentioned above.